src/fluent/stdout.h

```diff
@@ -46,7 +46,7 @@ class Stdout {
   }
 
  private:
-  std::set<std::tuple<std::string>> deferred_merge_;
+  std::vector<std::tuple<std::string>> deferred_merge_;
```

Comment:
I agree. But at line 46 of table.h, we are using make_move_iterator on begin(deferred_merge), which leads to a compiler error. Is the make_move_iterator just an optimization? If so can we just get rid of it?